src/Pos/Txp/Toil/Logic.hs

```diff
@@ -164,3 +172,39 @@ withTxId aux@(tx, _, _) = (hash tx, aux)
 
 applyTxToUtxo' :: MonadUtxo m => (TxId, TxAux) -> m ()
 applyTxToUtxo' (i, (t, _, d)) = Utxo.applyTxToUtxo (WithHash t i) d
+
+#ifdef WITH_EXPLORER
+modifyAddrHistory
```

Comment:
taks, these functions should be moved to `src/Pos/Txp/Toil/History` (or somewhere else), I suppose.
@gromakovsky what do you think?